exp-player/config/environment.js

```diff
@@ -2,5 +2,32 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+      moment: {
+          includeLocales: true
```

Comment:
For any front-end application, we should try to avoid having the user download more information than is needed.

Does this option cause *all* locales to be part of the final built app? As an easy test: if you tell ember-moment to only use en-us, is the file size from `ember build` the same size, or smaller, than if you use `includeLocales: true` (the "everything" option)?

https://github.com/stefanpenner/ember-moment#i18n-support